full/src/test/java/apoc/trigger/TriggerClusterTest.java

```diff
@@ -0,0 +1,109 @@
+package apoc.trigger;
+
+import apoc.util.TestContainerUtil;
+import apoc.util.TestUtil;
+import apoc.util.TestcontainersCausalCluster;
+import org.junit.AfterClass;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.neo4j.driver.types.Node;
+import org.neo4j.internal.helpers.collection.MapUtil;
+
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+
+import static apoc.util.TestUtil.isRunningInCI;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assume.assumeFalse;
+
+public class TriggerClusterTest {
+
+    private static TestcontainersCausalCluster cluster;
+
+    @BeforeClass
+    public static void setupCluster() {
+        assumeFalse(isRunningInCI());
+        TestUtil.ignoreException(() ->  cluster = TestContainerUtil
+                .createEnterpriseCluster(3, 1, Collections.emptyMap(), MapUtil.stringMap(
+                        "apoc.trigger.refresh", "100",
+                        "apoc.trigger.enabled", "true"
+                )),
+                Exception.class);
+        Assume.assumeNotNull(cluster);
+    }
+
+    @AfterClass
+    public static void bringDownCluster() {
+        if (cluster != null) {
+            cluster.close();
+        }
+    }
+
+    @Before
+    public void before() {
+        cluster.getSession().run("CALL apoc.trigger.removeAll()");
+        cluster.getSession().run("MATCH (n) DETACH DELETE n");
+    }
+
+
+    @Test
+    public void testTimeStampTriggerForUpdatedProperties() throws Exception {
+        cluster.getSession().run("CALL apoc.trigger.add('timestamp','UNWIND apoc.trigger.nodesByLabel($assignedNodeProperties,null) AS n SET n.ts = timestamp()',{})");
```

Comment:
wouldn't `$assignedLabels` make more sense here (and below)?

Reply:
Also, maybe for another PR, the short documentation of apoc.trigger.add still shows the old parameter syntax (https://neo4j.com/labs/apoc/4.0/overview/apoc.trigger/apoc.trigger.nodesByLabel/)

Reply:
oki